pkg/solidity-utils/contracts/helpers/TransientStorageHelpers.sol

```diff
@@ -106,7 +114,7 @@ library TransientStorageHelpers {
         lastElementSlot.tstore(address(0));
     }
 
-    // Uint256
+    // Uint256 values
```

Comment:
In other files, we used the pattern:

```
/*************************
         Uint256 values
*************************/
```

Should we adopt this pattern in that files, or that pattern in this file, or are them 2 separate things?

Reply:
I'd thought about that - we use the block headers everywhere else. These files were adapted from the old OZ code, which had this style. Should update to the block headers, I think.